molecule/default/molecule.yml

```diff
@@ -13,6 +13,10 @@ platforms:
     image: debian:stretch-slim
   - name: debian10
     image: debian:buster-slim
+  - name: debian11
+    image: debian:bullseye-slim
+  - name: kali
```

Comment:
Shouldn't Kali alphabetically go after Fedora and before Ubuntu?

Reply:
Yes, but @mcdonnnj did it this way in the skeleton; I assume this is because Kali is in the Debian family.  I decided that sorting by OS family and then sorting alphabetically within OS families was also reasonable.

Reply:
I didn't notice this in the skeleton- as long as we are doing it consistently, that is good enough for me.